### PR TITLE
CI Update

### PR DESCRIPTION
**Pull request by mairaw**
/cc @dend

### CODE REVIEW

**Comment by mairaw** [comment]
Adding @cartermp for F#. I'm going to add some comments about some of the things that I'm seeing different from MSDN, but I'm not sure which one is correct.

File: xml/Accessibility/AnnoScope.xml [hunk 1]
```diff
@@ -2,6 +2,8 @@
   <TypeSignature Language="C#" Value="public enum AnnoScope" />
   <TypeSignature Language="ILAsm" Value=".class public auto ansi sealed AnnoScope extends System.Enum" />
   <TypeSignature Language="DocId" Value="T:Accessibility.AnnoScope" />
+  <TypeSignature Language="VB.NET" Value="Public Enum AnnoScope" />
+  <TypeSignature Language="F#" Value="type AnnoScope = " />
```
Comment (mairaw) [inline xml/Accessibility/AnnoScope.xml]: MSDN: type AnnoScope

File: xml/Accessibility/CAccPropServices.xml [hunk 2]
```diff
@@ -2,6 +2,8 @@
   <TypeSignature Language="C#" Value="public interface CAccPropServices : Accessibility.IAccPropServices" />
   <TypeSignature Language="ILAsm" Value=".class public interface auto ansi abstract CAccPropServices implements class Accessibility.IAccPropServices" />
   <TypeSignature Language="DocId" Value="T:Accessibility.CAccPropServices" />
+  <TypeSignature Language="VB.NET" Value="Public Interface CAccPropServices&#xA;Implements IAccPropServices" />
+  <TypeSignature Language="F#" Value="type CAccPropServices = interface&#xA;    interface IAccPropServices" />
```
Comment (mairaw) [inline xml/Accessibility/CAccPropServices.xml]:
```
type CAccPropServices = 
    interface
        interface IAccPropServices
    end
```

**Comment by cartermp** [comment]
15k files 😱 

What are the relevant changes here?

**Comment by mairaw** [comment]
Not expecting you to review the 15k files @cartermp. This is adding F# and VB signatures to all the APIs. I've comment on the first two I was double checking with MSDN as an example.

File: xml/Accessibility/AnnoScope.xml (hunk 1 above)
Comment (cartermp) [reply xml/Accessibility/AnnoScope.xml]: In this case, the ` = ` is different, but still correct.  F# enums are defined like this:

``fsharp
type MyEnum =
    | Val1 = 1
    | Val2 = 2
    | Val3 = 3
```

MSDN showed these without the ` = `.  I'm not sure which is better or worse, so this is probably okay.

File: xml/Accessibility/CAccPropServices.xml (hunk 2 above)
Comment (cartermp) [reply xml/Accessibility/CAccPropServices.xml]: So this is a bit tricky.  The "right" way to do this in modern F# is as follows:

```fsharp
type CAccPropServices =
    interface IAccPropServices with
        member_.ClearHwndProps(hwnd, idObject, idChild, .....)
```

However, given that interface members are not listed in MSDN either, the best way to show this is probably like this:

```fsharp
type CAccPropServices =
    interface IAccPropServices with
        ...
```
Currently, it renders like this:

```fsharp
type CAccPropServices = interface
    interface IAccPropServices
```

Which is not correct.

**Comment by mairaw** [comment]
Closing for now.

**Pull request closed without merging**